internal/cmd/error.go

```diff
@@ -18,14 +18,14 @@ type Error struct {
 func (e Error) Error() string {
 	if e.OriginalError != nil {
 		if len(e.Message) == 0 {
-			return fmt.Sprintf("Internal error:\n%v", e.OriginalError)
+			return fmt.Sprintf("Internal error: %v", e.OriginalError)
```

Comment:
`Internal error` doesn't seem accurate here. The Error type is for user-facing errors is it not?

Reply:
The purpose of the Error is to allow for user-facing output, so any errors (internal or developer specified) can all be formatted consistently. Basically to wrap any errors (or create any) that can be printed on the CLI nicely